### DISCUSSION TITLE
idk

### DISCUSSION THREAD

**Discussion by NoahWantsDie** [NoahWantsDie/fnf-shits#11] [Announcements]
testing crap out with git hub i dont know what this is gonna do, looks like something i can post updates with

**Comment by NoahWantsDie** [comment]
looks like it

**Comment by DrakeIsBruh** [comment]
yo btw heres a knuckles exe chromatic (i noticed you didnt supply one)
https://drive.google.com/file/d/1h_nfxM1MOsj4P-AwVBh-eJ02LRnHHV1c/view?usp=sharing

**Reply by NoahWantsDie** [reply]
official?

**Reply by DrakeIsBruh** [reply]
Not sure, it's from a large library of chromatics on GitHub. I can add the link to it when I'm at home.

**Reply by DrakeIsBruh** [reply]
> official?

heres the link to all of them btw chief
https://github.com/Chimmie-mpeg/FNF-FANMADE-CHROMATIC-SCALES

**Comment by BirdoBirdo** [comment]
Hey, you did well on triple trouble, do you have a four-way fracture midi?
